index.bs

```diff
@@ -1696,22 +1696,80 @@ Team Representative in an Interest Group</h5>
-	feedback on the <a href="#ACCommunication">Advisory Committee discussion list</a>
-	or via other designated channels.
-
-	W3C <em class="rfc2119">may</em> begin work
-	on a Working Group or Interest Group charter
-	at any time.
+	W3C creates [=charters=] for [=chartered groups=] based on interest from the [=Members=] and [=Team=].
+	[=Charters=] are formally approved through an [=AC Review=] and [=W3C Decision=] (see [[#CharterReview]]).
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+	The [=Team=] <em class=rfc2119>may</em> send notice to the [=AC=]
+	when starting development of a new charter prior to the [=charter refinement=] phase.
+
+<h3 id="charter-initiation">
+Initiating Charter Refinement</h3>
+
+	Formal [=charter refinement=] (see below) is initiated
+	by the [=Team=] sending a [=charter review notice=]
+	to the [=Advisory Committee=],
+	to the public,
+	and, in the case of rechartering, to the affected [=Group=].
+
+	This <dfn>charter review notice</dfn> must include:
+	* A short summary of the proposal.
+	* The location of the [=charter draft=], which must be public.
+	* How to participate in the discussion of this [=charter draft=] and where to file issues.
+	* The expected duration of the [=charter refinement=] phase,
+		which must not be less than 28 days,
+		and should not be more than 6 months.
+	* Who the [=Chartering Facilitator=] is.
+
+	The [=Team=] is responsible for initiating [=charter refinement=]
+	at its discretion, in consideration of discussions with the community.
+	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
+	that the [=Team=] initiate [=charter refinement=].
+	The Team <em class=rfc2119>may</em> deny such a request
+	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission.
+	This rejection is a [=Team Decision=] which can be <a href="#registering-objections">Formally Objected to</a>.
+
+<h3 id="charter-development" oldids="WGCharterDevelopment">
+Charter Refinement</h3>
+
+	During <dfn>charter refinement</dfn>,
+	the W3C community further develops the [=charter draft=]
+	with the goal of achieving [=consensus=] on the proposal.
+	The <dfn>Chartering Facilitator</dfn>
+	acts as [=Chair=] for the [=charter refinement=] process.
+
+	Note: The [=Chartering Facilitator=] is not necessarily (and often is not) the [=Chair=] of the [=chartered group|group=] being chartered.
+
+	During [=charter refinement=]:
+	* All issues filed against the [=charter draft=] must be [=formally addressed=],
+		and their resolutions tracked in a disposition of comments.
+	* [=Wide review=] of the [=charter draft=] is initiated and completed.
+	* Decisions are made as  [=group decisions=] of all individuals participating in this process.
```

Comment:
Who are the individuals participating in this process? How does one become one of them?

Reply:
Anybody who bothers participating? By participating? Typically that would be filling github issues, or responding to those that have been filled already, or responding to CfCs, or showing up to  any meeting organized, or engaging in any other way listed in the charter review notice as the way to participate.

Reply:
Process CG resolved: Consensus is based on anyone participating.